services/hydra/default.nix

```diff
@@ -140,13 +140,15 @@ in
       buildMachines = [
         {
           hostName = "localhost";
-          systems = [ "x86_64-linux" "builtin" ];
+          systems = [ "x86_64-linux" "aarch64-linux" "builtin" ];
           maxJobs = 8;
           supportedFeatures = [ "nixos-test" "big-parallel" "kvm" ];
         }
       ];
     };
 
+    boot.binfmt.emulatedSystems = [ "aarch64-linux" ];
```

Comment:
We have build04 as a remote builder with native aarch64. Maybe just use that. It's an oracle arm machine.